### DISCUSSION TITLE
How do I solve the issue of `not being able to open in lower versions of Word` (compatibility issue)

### DISCUSSION THREAD

**Discussion by Vinsea** [dolanmiu/docx#2414] [Show and tell]
The background is that the file I generated through `docx tool` cannot be opened in Word 2007. 
I would like to share [my resolution steps](https://vinxea.com/front-end/docx-js-issues) and hope it can help everyone.

> Sorry, foreign friends, you may need translation tools

feel free to leave messgae and discuss!!!

some similar issues：
- #2175 
- #1113

**Comment by dolanmiu** [comment]
Thank you for sharing the resolution steps
I will look into adding these into docx officially to prevent this workaround

**Reply by Vinsea** [reply]
its my honor
